spec/lib/jobs/local-catalog-spec.js

```diff
@@ -0,0 +1,179 @@
+// Copyright 2015, EMC, Inc.
+/* jshint node:true */
+
+'use strict';
+
+describe('Local Catalog Job', function () {
+    var LocalCatalogJob;
+    var Promise;
+    var uuid;
+    var mockWaterline = {
+        nodes: {},
+        catalogs: {}
+    };
+
+    before(function() {
+        helper.setupInjector(
+            _.flatten([
+                helper.require('/lib/jobs/base-job'),
+                helper.require('/lib/jobs/local-catalog'),
+                helper.require('/lib/utils/job-utils/command-parser'),
+                helper.di.simpleWrapper(mockWaterline, 'Services.Waterline')
+            ])
+        );
+
+        Promise = helper.injector.get('Promise');
+        LocalCatalogJob = helper.injector.get('Job.Local.Catalog');
+        uuid = helper.injector.get('uuid');
+    });
+
+    describe('input validation', function(){
+        var job;
+
+        beforeEach('Local catalog job input validation', function(){
+            var options = {
+                commands: ['test_command'],
+                acceptedResponseCodes: [1]
+            };
+            job = new LocalCatalogJob(options, { target: 'bc7dab7e8fb7d6abf8e7d6ac' }, uuid.v4());
+            mockWaterline.nodes.findByIdentifier = function(){};
+
+            this.sandbox = sinon.sandbox.create();
```

Comment:
Just as @benbp  mentioned in another PR "This was an error that I put in a bunch of tests a while back. This statement can actually go in the before block instead of the beforeEach block, since we only need to create the sandbox once (and will save us test execution time)."